crates/interledger-settlement-engines/src/engines/ethereum_ledger/eth_engine.rs

```diff
@@ -563,20 +563,33 @@ where
                     tx.gas = data[1];
                     tx.nonce = data[2];
 
+                    trace!(
+                        "Gas required for transaction: {}, gas price: {}",
+                        data[1],
+                        data[0]
+                    );
+
                     let signed_tx = signer.sign_raw_tx(tx.clone(), chain_id); // 3
                     let action = move || {
+                        trace!("Sending tx to Ethereum: {}", hex::encode(&signed_tx));
                         web3.eth() // 4
+                            // TODO use send_transaction_with_confirmation
```

Comment:
Why? We send the transaction and it should be guaranteed it will eventually be confirmed. The receiver only cares about the confirmations, we care that it gets broadcast

Reply:
I was running into issues where this call would fail silently. It may have just been Infura rate limiting me because of https://github.com/emschwartz/interledger-rs/issues/203#issuecomment-521385170 but the SE would log "Transaction submitted..." but then it never showed up on the testnet.

Reply:
I could be convinced that this is unnecessary overhead but it was really difficult to figure out what was going wrong when the logs said it had been submitted but then it never made it into a block

Reply:
I do expect that this has to do with infura. We should add instructions for connecting to testnet/mainnet via geth/parity as well.